packages/core-sdk/src/meta-tx/handler.ts

```diff
@@ -600,7 +600,7 @@ export async function signMetaTxPreMint(
 
 export async function signMetaTxSetApprovalForAll(
   args: BaseVoucherMetaTxArgs & {
-    operator: string;
```

Comment:
we should keep "operator" here because it matches the standard terminology of ERC721 https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/dd8ca8adc47624c5c5e2f4d412f5f421951dcc25/contracts/token/ERC721/ERC721Upgradeable.sol#L141